src/App.css

```diff
@@ -37,10 +42,11 @@
   }
 }
 
-.nav{
-  height: 11vh;
+.center{
+  text-align: center;
 }
 
-.signup{
-  margin: 5vh;;
+a{
+  text-decoration: none;
+  color: white;
```

Comment:
In general ay 7aga that you want to do with classes a7san t3melha as a styled component, w momken t5alleh global component in the components directory and you can use it anywhere in the project

Reply:
7abeeby <3